packages/terra-docs/src/terra-dev-site/tool/DocumentationComponents/Button.tool.mdx

```diff
@@ -0,0 +1,37 @@
+import { Badge } from '@cerner/terra-docs/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+
+import Button from '../example/Button?dev-site-example';
+
+import PropsTable from '@cerner/terra-docs/src/Button?dev-site-props-table';
+
+<Badge />
+
+# terra-docs Button
+
+A a simple button for documentation examples and tests.
```

Comment:
```suggestion
A simple button for documentation examples and tests.
```

Reply:
Should we add some background info on why this button should be used over terra-button? I'm not sure consumers would understand why this could be better or when this should be used.